src/app/shared/components/dynamic-form/dynamic-form.component.html

```
@@ -0,0 +1,103 @@
+      <p-floatlabel variant="on" *ngIf="field.type !== 'password'">
+        <label [for]="field.name">{{ field.label | titlecase }}</label>
+
+        <!-- Input Text Field -->
+        <input
+          *ngIf="field.type === 'inputText'"
+          pInputText
+          [id]="field.name"
+          [formControlName]="field.name"
+          [class]="field.styleClass || ''"
+          [value]="form.get(field.name)?.value"
+          class="form-control w-full md:w-1/2 lg:w-1/3" />
```

HTMLHint (1.5.0)

[warning] The id and class attribute values must be in lowercase and split by a dash.

(id-class-value)